launch.go

```diff
@@ -20,20 +23,36 @@ import (
 // VERSION gets set by the build script via the LDFLAGS
 var VERSION string
 
-type scmPath struct {
-	Host   string
-	Org    string
-	Repo   string
-	Branch string
-}
-
 var mkdirAll = os.MkdirAll
 var stat = os.Stat
 var gitSetup = git.Setup
 var open = os.Open
 var executorRun = executor.Run
 var writeFile = ioutil.WriteFile
 
+var cleanExit = func() {
+	os.Exit(0)
+}
+
+// exit sets the build status and exits successfully
+func exit(status screwdriver.BuildStatus, api screwdriver.API) {
```

Comment:
I assume it would be a panic in that case, and the recoveries in launch and main should deal with them.